x-pack/plugin/sql/src/main/antlr/SqlBase.g4

```diff
@@ -235,6 +236,10 @@ castExpression
 castTemplate
     : CAST '(' expression AS dataType ')'
     ;
+
+builtinDateTimeFunction
+    : name=CURRENT_TIMESTAMP ('(' precision=INTEGER_VALUE? ')')?
```

Comment:
Wouldn't be simpler if we just define the `CURRENT_TIMESTAMP` and let the `CURRENT_TIMESTAMP()` follow the normal path of a function resolution?

Reply:
It would but it doesn't work since antlr will pick up the token and then complain about the extra `(`:
`mismatched input '(' expecting {<EOF>, 'AND', 'BETWEEN', 'IN', 'IS', 'LIKE', 'NOT', 'OR', 'RLIKE', '=', '<=>', NEQ, '<', '<=', '>', '>=', '+', '-', '*', '/', '%'}`

Reply:
ah I see, didn't cross my mind, thank you!